src/HTSPTypes.h

```diff
@@ -73,6 +73,21 @@ enum eSubscriptionWeight {
   SUBSCRIPTION_WEIGHT_SERVERCONF = 0,
 };
 
+enum eSubsriptionState
```

Comment:
Can you move this to `Subscription.h`? I want to get rid of `HTSPTypes.h` in the long run.